spec/requests/report_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
```

Comment:
As far as I understand, when you generate a controller (same with models) it creates a spec file that goes into spec/controller/controlers_name, in this case I am not sure what this spec/requests/ path corresponds to, did you create it by hand? was it automatically created with rails generate controller reports? or you used something like rails generate rspec:controller reports?

Reply:
Yes, the controllers already existed before the spec files were generated. `rails generate rspec:controller reports` is exactly what I did :)

Reply:
Then... LGTM! (hopefully we will be seeing this a lot in the future haha)